### PR TITLE
[Feat] - Add goversioninfo

### PR DESCRIPTION
**Pull request by jenaye**

### CODE REVIEW

**Comment by jenaye** [comment]
image

image

File: loader/aes.go [hunk 1]
```diff
@@ -20,6 +20,9 @@ func (a Aes) Load(content, key []byte) ([]byte, error) {
 		"key": func() string {
 			return string(key)
 		},
+		"base_path": func() string {
```
Comment (TomChv) [inline loader/aes.go:23]: It shall be `basePath` or `basepath`

File: loader/chacha20.go [hunk 2]
```diff
@@ -20,6 +20,9 @@ func (a ChaCha20) Load(content, key []byte) ([]byte, error) {
 		"key": func() string {
 			return string(key)
 		},
+		"base_path": func() string {
```
Comment (TomChv) [inline loader/chacha20.go:23]: It shall be `basePath` or `basepath`

File: loader/xor.go [hunk 3]
```diff
@@ -20,6 +20,9 @@ func (x Xor) Load(content, key []byte) ([]byte, error) {
 		"key": func() string {
 			return string(key)
 		},
+		"base_path": func() string {
```
Comment (TomChv) [inline loader/xor.go:23]: It shall be `basePath` or `basepath`

File: loader/loader.go [hunk 4]
```diff
@@ -31,6 +32,10 @@ const (
 	tmpDir  = "/tmp/221b-compile"
 )
 
+var (
+	base_path, _ = os.Getwd()
```
Comment (TomChv) [inline loader/loader.go:36]: It shall be `basePath` or `basepath`

File: templates/aes.go [hunk 5]
```diff
@@ -1,6 +1,7 @@
 package templates
 
 var AesTmpl = `
+//go:generate goversioninfo -icon={{ base_path }}/misc/pwned.ico -manifest={{ base_path }}/misc/goversioninfo.exe.manifest
```
Comment (TomChv) [inline templates/aes.go:4]: It shall be `basePath` or `basepath`

File: templates/chacha20.go [hunk 6]
```diff
@@ -1,6 +1,7 @@
 package templates
 
 var ChaCha20Tmpl = `
+//go:generate goversioninfo -icon={{ base_path }}/misc/pwned.ico -manifest={{ base_path }}/misc/goversioninfo.exe.manifest
```
Comment (TomChv) [inline templates/chacha20.go:4]: It shall be `basePath` or `basepath`

File: templates/xor.go [hunk 7]
```diff
@@ -1,6 +1,7 @@
 package templates
 
 var XorTmpl = `
+//go:generate goversioninfo -icon={{ base_path }}/misc/pwned.ico -manifest={{ base_path }}/misc/goversioninfo.exe.manifest
```
Comment (TomChv) [inline templates/xor.go:4]: It shall be `basePath` or `basepath`

**Review by TomChv — COMMENTED** [review]
Left 1 comment, LGTM otherwise

**Pull request merged**